imagination_bank/main.py

```diff
@@ -1 +1,71 @@
-pass
+import json
+def main():
+    account_name,account_balance,answer=create_account()
+    account_name,account_balance=login_user(account_name,account_balance,answer)
+    while True:
+        choice=interact_with_user()
+        if choice in ["Q","q"]:
+            exit()
+        elif choice in ["U","u"]:
+            print(f"Username is: {account_name}")
+            print()
+
+
+
+
+def create_account():
+    account_list=[]
+    print("Hello And Welcome To RealMasterChief's Bank, The Best In The World Of Online Banking.")
+    while True:
+        answer=input("Do you have an account already? [Y/N]")
+        answer=answer.upper()
+        if answer not in ["Y","N"]:
+            print("Input not recognized. Please try again.")
+            continue
+        if answer=="Y":
+            break
+        print("It's time to create an account!")
+        account_name=input("Please Enter an account name: ")
+        account_balance=0
+        with open('account.json','w') as p:
+            json.dump([[account_name,account_balance]],p)
```

Comment:
This only saves one account, and it overwrites the existing account, if there was one.
Let's reduce the complexity on this PR by not using json yet, and just building a system that works in memory.
So a continuous loop that only exits when you use control-c in the terminal.